### ISSUE TITLE
Fix date/time on Infrastructure Node Details page

### ISSUE THREAD

**Issue by jonong1972** [chef/automate#1705]
## Goal
Consistent date/time display across automate, that displays the day of the month, before the month.

## Overview
Currently, the date/time format on some pages does not match the consistency pattern we have developed. Specifically, it should be displayed as...
`<Day of week:word as abbreviation>, <Day: num, prepend w/ 0> <Month:word as abbreviation> <Year: num> <Time: 00:00:00> UTC`
e.g., `Wed, 03 Jul 2019 17:08:53 UTC`

#### Before
![Screen Shot 2019-10-02 at 10 28 19 AM](https://user-images.githubusercontent.com/1316587/66067080-b1a9cf00-e4ff-11e9-8133-6fb51f13efa1.png)

#### After
![Screen Shot 2019-10-02 at 10 27 43 AM](https://user-images.githubusercontent.com/1316587/66066979-7b6c4f80-e4ff-11e9-8066-1210bd8e5bab.png)

#### Before
![Screen Shot 2019-10-02 at 10 28 35 AM](https://user-images.githubusercontent.com/1316587/66067126-cc7c4380-e4ff-11e9-9c30-f665cf994af8.png)

#### After
![Screen Shot 2019-10-02 at 10 27 58 AM](https://user-images.githubusercontent.com/1316587/66067301-1fee9180-e500-11e9-89c2-9941fc7e97d1.png)

#### Before
![Screen Shot 2019-10-02 at 10 28 26 AM](https://user-images.githubusercontent.com/1316587/66067175-e74eb800-e4ff-11e9-9f4c-8897d35c4f8b.png)
#### After
*NOTE:  It would be sweet if we used the same look and feel as we do for the compliance history. I created a separate ticket for this - https://github.com/chef/automate/issues/1712

![Screen Shot 2019-10-02 at 10 27 52 AM](https://user-images.githubusercontent.com/1316587/66067000-88893e80-e4ff-11e9-803c-114b643b36af.png)

**Comment by susanev** [comment]
@jonong1972 for the ranges, im pretty sure you only need to say `UTC` once, but you may want to check with the docs team

**Comment by susanev** [comment]
im thinking the run duration should not have seconds either, im kinda wondering if that should be a second case that leverages the work with relative time that @apriofrost has done

**Comment by jonong1972** [comment]
@susanev, I talked with the Docs Team and they had suggested that we put the UTC in a (UTC+8), which i added above. We can discuss with the UX team. I will bring it up in our Wed sync. or when you get back.

**Comment by jonong1972** [comment]
After talking with the UX team on the inclusion of the timezone offset, we have decided to remove it for now due to running into the problem of grabbing the local time and adjusting the offset. There needs to be more research done, but in spirit of this ticket we are going forward without.

There were some questions about the Run Duration formatting, but we decided the problem was the labeling and the way it was displayed.  I created another ticket for this issue. https://github.com/chef/automate/issues/1768
